src/lib/agentforce/agentforce-code-action-provider.ts

```diff
@@ -28,7 +28,7 @@ export class AgentforceCodeActionProvider implements vscode.CodeActionProvider {
         const codeActions: vscode.CodeAction[] = [];
         const filteredDiagnostics: CodeAnalyzerDiagnostic[] = context.diagnostics
             .filter(d => d instanceof CodeAnalyzerDiagnostic)
-            .filter(d => range.contains(d.range) && A4D_SUPPORTED_RULES.has(d.violation.rule));
+            .filter(d => !d.isStale() && range.contains(d.range) && A4D_SUPPORTED_RULES.has(d.violation.rule));
```

Comment:
This stale behavior change is good, as it won't show as a quick fix option anymore. It's probably worth calling out in the docs too @jshackell-sfdc